publish/views/asset.py

```diff
@@ -34,11 +35,11 @@ class Meta(AssetSerializer.Meta):
 
 
 class AssetFilter(filters.FilterSet):
-    path_prefix = filters.CharFilter(lookup_expr='startswith', field_name='path')
+    path = filters.CharFilter(lookup_expr='exact', field_name='path')
```

Comment:
I don't think you need to explicitly define a `path =` filter at all. The fact that we've requested `fields = ['path']` *should* be enough for it to auto-generate a standard filter from the model.

Reply:
Good point, I'll change it and make sure it behaves like we expect.

Reply:
e9a1aaf